server.js

```diff
@@ -77,16 +77,10 @@ const config = require('nconf')
 	.file('environment', { file: `config/${process.env.NODE_ENV}.json` })
 	.file('defaults', { file: 'config/default.json' });
 
-//DB
-const mongoose = require('mongoose');
-mongoose.connect(config.get('mongodb_uri') || config.get('mongolab_uri') || 'mongodb://localhost/naturalcrit',
-	{ retryWrites: false });
-mongoose.connection.on('error', (err)=>{
-	console.log('Error : Could not connect to a Mongo Database.');
-	console.log('        If you are running locally, make sure mongodb.exe is running.');
-	console.log(err);
-	throw 'Can not connect to Mongo';
-});
+// DB
+const DB = require('./server/db.js');
+// FIXME: consider implementing error hanlding for a Promise returned by connect
```

Comment:
Note: `node` already handles errors here, which looks like this:
```
$ NODE_ENV=local PORT=8888 MONGODB_URI=invalid node server.js
server on port:8888
Could not connect to a Mongo Database.
MongoParseError: Invalid connection string "invalid"
    at new ConnectionString (github.com\naturalcrit\homebrewery\node_modules\mongodb-connection-string-url\lib\index.js:68:19)
    at parseOptions (github.com\naturalcrit\homebrewery\node_modules\mongodb\lib\connection_string.js:213:17)
    at new MongoClient (github.com\naturalcrit\homebrewery\node_modules\mongodb\lib\mongo_client.js:62:63)
    at github.com\naturalcrit\homebrewery\node_modules\mongoose\lib\connection.js:779:16
    at new Promise (<anonymous>)
    at NativeConnection.Connection.openUri (github.com\naturalcrit\homebrewery\node_modules\mongoose\lib\connection.js:776:19)
    at github.com\naturalcrit\homebrewery\node_modules\mongoose\lib\index.js:332:10
    at promiseOrCallback (github.com\naturalcrit\homebrewery\node_modules\mongoose\lib\helpers\promiseOrCallback.js:10:12)
    at Mongoose._promiseOrCallback (github.com\naturalcrit\homebrewery\node_modules\mongoose\lib\index.js:1158:10)
    at Mongoose.connect (github.com\naturalcrit\homebrewery\node_modules\mongoose\lib\index.js:331:20)
If you are running locally, make sure mongodb.exe is running.
node:internal/process/promises:246
          triggerUncaughtException(err, true /* fromPromise */);
          ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "undefined".] {
  code: 'ERR_UNHANDLED_REJECTION'
}
```

My comment is kind of asking about whether we want to have anything on top of that, like control over how error message is printed, control over actions which are done on error (second attempt to connect, for example), etc.